typings/index.d.ts

```diff
@@ -128,6 +128,31 @@ import {
   RawWidgetMemberData,
 } from './rawDataTypes';
 
+export {
+  ActivityTypes,
+  ApplicationCommandOptionTypes,
+  ApplicationCommandPermissionTypes,
+  ApplicationCommandTypes,
+  ChannelTypes,
+  DefaultMessageNotificationLevels,
+  ExplicitContentFilterLevels,
+  InteractionResponseTypes,
+  InteractionTypes,
+  InviteTargetType,
+  MembershipStates,
+  MessageButtonStyles,
+  MessageComponentTypes,
+  MFALevels,
+  NSFWLevels,
+  OverwriteTypes,
+  PremiumTiers,
+  PrivacyLevels,
+  StickerFormatTypes,
+  StickerTypes,
+  VerificationLevels,
+  WebhookTypes,
+} from './enums';
```

Comment:
Not really sure but shouldn't this work too?
```suggestion
export * from './enums';
```

Reply:
Yeah that works, probably a better solution as it requires less maintenance/upkeep.